app/controllers/measurements_controller.rb

```diff
@@ -1,5 +1,12 @@
 class MeasurementsController < ApplicationController
   before_action :set_measurement, only: [:show, :edit, :update]
+  after_action :send_user_mail, only: [:create]
+
+  def send_user_mail
```

Comment:
This method should be abstracted to a helper file in future - added to trello board